rocky/reports/report_types/dns_report/report.html

```diff
@@ -153,13 +153,12 @@ <h3>{% translate "Other findings found" %}</h3>
                     </tr>
                     <tr class="expando-row">
                         <td colspan="5">
-                            <h3>{{ info.finding_type }}</h3>
                             <h4>{% translate "Findings information" %}</h4>
                             <dl>
                                 <div>
                                     <dt>{% translate "Finding" %}</dt>
                                     <dd>
-                                        {{ info.finding_type.id }}
+                                        {{ info.finding_type.primary_key }}
```

Comment:
Should we not show the human readable here too?

Reply:
Would be nice, but this isn't possible here